Assignment2/BasilisKing-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,106 @@
+{
+"@context":{
+  "upm": "http://etsiinfdb.es/",
+  "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+  "fullName": "rdfs:label",
+  "hometown": "rdfs:label",
+  "age": "rdfs:label",
+  "almaMater": "rdfs:label",
+  "parents" : "upm:prog",
+  "mother": "rdfs:label",
+  "father": "rdfs:label",
+  "name": "rdfs:label",
```

Comment:
Syntactically is ok, but you have to define each attribute with a specific vocabulary attribute name because when you want to retrieve, for example your age, if you query "rdfs:label" you will not be able to distinguish what data you ask for. If you have your own vocabulary you can use it to define your attributes e.g. "name":"yourvocab:name", "age":"yourvocab:age", etc.